hbase/Dockerfile

```diff
@@ -87,6 +83,40 @@ if [ "${DELETE_CACHES}" = "true" ] ; then
 fi
 EOF
 
+# hadolint ignore=DL3006
+FROM stackable/image/java-devel AS opa-authorizer-builder
+
+ARG OPA_AUTHORIZER
+ARG DELETE_CACHES
+
+USER stackable
+WORKDIR /stackable
+
+RUN --mount=type=cache,id=maven-opa,uid=1000,target=/stackable/.m2/repository <<EOF
+
+###
+### OPA Authorizer (only for 2.6 upwards)
+###
+if [[ -n "$OPA_AUTHORIZER" ]]; then
+    git clone --depth 1 --branch "$OPA_AUTHORIZER" https://github.com/stackabletech/hbase-opa-authorizer.git
+    mvn \
+      --quiet \
```

Comment:
Can you remove the quiet flag? I like to see at least a bit what's going on

Reply:
Fixed https://github.com/stackabletech/docker-images/pull/701/commits/0afda96d8ab0f590d0c825e23972bab47181bd77